src/components/Button.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+/**
+ * @param {string} text 버튼 내부 텍스트
+ * @param {string} type 버튼 타입 (button, submit)
+ * @param {string} error 표시할 에러 메시지
+ * @param {() => void } handleClick 버튼 클릭시 실행할 이벤트, Route 기능의 경우 history.push와 같은 함수를 이용합니다.
```

Comment:
```suggestion
 * @param { () => void } handleClick 버튼 클릭시 실행할 이벤트, Route 기능의 경우 history.push와 같은 함수를 이용합니다.
```
이렇게 하시던

```suggestion
 * @param {() => void} handleClick 버튼 클릭시 실행할 이벤트, Route 기능의 경우 history.push와 같은 함수를 이용합니다.
```

이렇게 하시던.. 스페이스는 양쪽 맞춰서..

Reply:
아앗... 공백 제거할게요

Reply:
(참고로 위에 Commit suggestion 누르면 바로 커밋되여 ㅊㄱㅊㄱ)